### PR TITLE
Update dev env with new docker images

### PR DESCRIPTION
**Pull request by bersace**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
+!temboard.dev.conf
 *.egg-info/
 build/
 dist/
 site/
 temboard-agent/test/test/configuration.py
-# We provide override. Base compose file comes from
-# https://github.com/dalibo/docker/blob/master/temboard/docker-compose.yml
-docker-compose.yml
+docker-compose.override.yml
```
Comment (pgiraud) [inline .gitignore]: Do you want to keep this in .gitignore? It looks like it is removed now.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,20 +3,14 @@
 Setup quickly a development environment with Docker & Compose.
 
 ``` console
-$ wget https://raw.githubusercontent.com/dalibo/docker/master/temboard/docker-compose.yml
-$ docker-compose up
+$ mkvirtualenv temboard
```
Comment (pgiraud) [inline CONTRIBUTING.md]: I would replace this by something saying "Create a python virtualenv and activate it"

File: .gitignore (hunk 1 above)
Comment (bersace) [inline .gitignore]: I prefer to let developer add customization to compose.yml in `override` like other instance/agent, etc. Especially once agent will auto register.

Comment (pgiraud) [inline .gitignore]: It makes sense. Thanks.

**Comment by pgiraud** [comment]
LGTM.

File: temboardui/utils.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        try:
+            conn = engine.connect()
+            conn.close()
+            break
+        except Exception as e:
+            if i == 9:
+                raise
+            logger.warn("Failed to connect to database: %s", e)
+            logger.info("Retrying in %ss.", i)
+            sleep(i)
```
Comment (pgiraud) [inline temboardui/utils.py]: The sleep duration increases at each loop. Is it done on purpose?
```
Failed to connect to database: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Retrying in 0s.
Failed to connect to database: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Retrying in 1s.
Failed to connect to database: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Retrying in 2s.
Failed to connect to database: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Retrying in 3s.
Failed to connect to database: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Retrying in 4s.
Failed to connect to database: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Retrying in 5s.
Failed to connect to database: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Retrying in 6s.
Failed to connect to database: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Retrying in 7s.
Failed to connect to database: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Retrying in 8s.
Connection to the database failed: (psycopg2.OperationalError) could not connect to server: Connection refused
  Is the server running on host "0.0.0.0" and accepting
  TCP/IP connections on port 5432?
Please check your configuration.
```

Comment (bersace) [inline temboardui/utils.py]: yes. It's a linear backoff. It allows to quickly retry if postgres is there while avoiding tons of logs if postgres is waiting.

The long time can be when you launch temboard and trigger compose up afterward, (you read the doc, open another terminal, etc.)

**Pull request merged**